### ISSUE TITLE
[Bug] Doc in read only - Save as doesn't tell it can't save 

### ISSUE THREAD

**Issue by nath5394** [ONLYOFFICE/DesktopEditors#212]
**Do you want to request a *feature* or report a *bug*?**
A bug

**What is the current behavior?**
I open a doc which is in read only mode for the current user, modify it, click save as, same name, ok to replace, and no message after

**If the current behavior is a bug, please provide the steps to reproduce and if possible a minimal demo of the problem.**
See before :arrow_up: 

**What is the expected behavior?**
I except to have an error message telling me the file can't be replaced because it is in read only, and I should change directory or name. 

**Which versions of DesktopEditors, and which OS are affected by this issue? Did this work in previous versions of DesktopEditors?**
ArchLinux, DesktopEditors 5.2.83.480.

**Comment by Rita-Bubnova** [comment]
Hello, @nath5394.
I'm not sure I understand you. Could you please describe in more detail the steps to reproduce the problem?
Do you open a file stored on the portal or on a local disk?
If from local disk - do you change some file system permissions to it?

**Comment by nath5394** [comment]
Hello, yes here are the steps I went trough to see this bug : 

- From firefox I click on a link to download a docx file
- In the download window I choosed "Open in Onlyoffice" and not download
- The file is opened in Onlyoffice
- At this time the file is in a firefox tmp directory, on the local drive, the file has read only rights
- I edit it in Onlyoffice
- I click save as
- The suggested dir is the firefox tmp directoy, I should normally choose another dir, but initially I click ok directly
- I am asked if I am ok to replace the file
- I click OK (*)
- I quit Onlyoffice
- In fact the file has not been written by Onlyoffice, I loosed my work (not a big problem in my cas, just a few modifications) 

(*) The bug is here I think : at this step I think Onlyoffice should warn me it can not save because the file is read only.

I hope this message is clear, feel free to tel me if it is the the case

**Comment by Rita-Bubnova** [comment]
Thank you. 
I managed to reproduce bug -  issue 41024 in our internal issue tracker.

**Comment by nlgranger** [comment]
The label enhancement on this is wrong as it leads to lost work in a very frequent use-case (document downloaded from webmail or mail client). 
Besides it doesn't seem to require extensive changes (just a check on file permissions and a redirection of save to save as), so is there any way to bump the priority on this?

**Comment by ShockwaveNN** [comment]
@nlgranger yeah, agree. marked like `enchancement` by error, this is defenetly a bug

I'll make a bump to our dev team

**Comment by kyylvisa** [comment]
Still an issue in 2025. File silently fails to save. File is also opened as read only when permissions are normal/not the issue. Had to recreate the file for it to be editable, perms did not change, this was verified by comparing perms using ls -al

```
-rw-r--r--  1 helly helly   72 Sep 27 18:58 .~lock.Foods.xlsx#
-rw-r--r--  1 helly helly   72 Sep 28 11:03 .~lock.modFoodsDetails.xlsx#
-rw-r--r--  1 helly helly 9923 Sep 28 11:03 modFoodsDetails.xlsx
```
Perhaps the issue is that onlyoffice did not properly remove the lock file, preventing editing/saving.

Only noticeable difference was that file was originally created by flatpak version. Neither flatpak nor [onlyoffice-bin](https://aur.archlinux.org/packages/onlyoffice-bin) was able to edit the flatpak made version.

Bit of a rant but this is a huge bug to seemingly not get prioritized. While my data loss is annoying, it's not critical but this is definitely a blocker for this software to ever be considered viable in a professional or commercial environment.
